static/app/components/events/interfaces/crashContent/exception/actionableItemsUtils.tsx

```diff
@@ -0,0 +1,275 @@
+import {EventErrorData} from 'sentry/components/events/errorItem';
+import findBestThread from 'sentry/components/events/interfaces/threads/threadSelector/findBestThread';
+import getThreadException from 'sentry/components/events/interfaces/threads/threadSelector/getThreadException';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {
+  CocoaProcessingErrors,
+  GenericSchemaErrors,
+  HttpProcessingErrors,
+  JavascriptProcessingErrors,
+  NativeProcessingErrors,
+  ProguardProcessingErrors,
+} from 'sentry/constants/eventErrors';
+import {tct} from 'sentry/locale';
+import {Project} from 'sentry/types';
+import {DebugFile} from 'sentry/types/debugFiles';
+import {Image} from 'sentry/types/debugImage';
+import {EntryType, Event, ExceptionValue, Thread} from 'sentry/types/event';
+import {defined} from 'sentry/utils';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import {semverCompare} from 'sentry/utils/versions';
+import {projectProcessingIssuesMessages} from 'sentry/views/settings/project/projectProcessingIssues';
+
+const MINIFIED_DATA_JAVA_EVENT_REGEX_MATCH =
+  /^(([\w\$]\.[\w\$]{1,2})|([\w\$]{2}\.[\w\$]\.[\w\$]))(\.|$)/g;
+
+export type ActionableItemTypes =
+  | JavascriptProcessingErrors
+  | HttpProcessingErrors
+  | GenericSchemaErrors
+  | ProguardProcessingErrors
+  | NativeProcessingErrors;
+
+export const ActionableItemWarning = [
+  ProguardProcessingErrors.PROGUARD_MISSING_LINENO,
+  NativeProcessingErrors.NATIVE_MISSING_OPTIONALLY_BUNDLED_DSYM,
+  GenericSchemaErrors.FUTURE_TIMESTAMP,
+  GenericSchemaErrors.CLOCK_DRIFT,
+  GenericSchemaErrors.PAST_TIMESTAMP,
+  GenericSchemaErrors.VALUE_TOO_LONG,
+];
+
+interface BaseActionableItem {
+  data: any;
+  message: string;
+  type: ActionableItemTypes;
+}
+interface ProguardMissingLineNoError extends BaseActionableItem {
+  type: ProguardProcessingErrors.PROGUARD_MISSING_LINENO;
+}
```

Comment:
the plan is for some of these to have custom data? right now they don't have much value

Reply:
yup, i was aiming to do something similar to what you did for the source map alerts, but as of now , we only have the event errors so this will be better utilized as we add more to this widget.